episodes/filters-and-thresholding.md

```diff
@@ -676,7 +677,7 @@ spread of pixel values in both the background and your class of interest. In
 effect, it's trying to find two peaks in the image histogram and place a 
 threshold in between them.
 
-This means it's only suitable for images that show two clear peaks in their 
+This means it's best suitable for images that show two clear peaks in their 
```

Comment:
```suggestion
This means it's most suitable for images that show two clear peaks in their 
```
I think `most` fits here better? Otherwise, something like 'This means it's best suited to images that show two clear peaks...'